src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -3114,7 +3116,7 @@ void AutomationInstrumentClipView::handleMultiPadPress(ModelStackWithTimelineCou
 			                            effectiveLength, false);
 
```

Comment:
I don't think this caused any bugs before, but the statement was wrong. the subtraction of kParamNodeWidth should've been outside the std::min() brackets.